libpod/image/tree.go

```diff
@@ -0,0 +1,138 @@
+package image
```

Comment:
Can we move this to somewhere in pkg/? It doesn't seem much like an API call, given it only returns a string.

Reply:
I suggest to wait for @mtrmac's new package. We may be able to integrate it there.

Reply:
By "wait" I mean we could merge it as is now and let Miloslav pick it up in his new package.

Reply:
Hm. It's just libpod/image, not libpod itself, so sure

Reply:
Definitely don’t wait for me, I’m still mostly in learning phase, and I can adjust for whatever happens here.

(Jumping into what I’m sure was a well-considered decision, I’d instinctively prefer to primarily have a well-typed API that returns the structure, and keep the text formatter in the CLI (allowing, at least in principle, a nice GUI to be built on top, for example). OTOH, I guess that exposing the complete tree structure over Varlink/whatever would be a lot of work and pain — and for no benefit at least now that the only consumer is the CLI.)

Reply:
That's my big hold-up as well - I'd love to return a data structure that represents the tree, with a `.String()` that returns the output. I'm willing to defer that to followup work, though.

Reply:
(Honestly, though, I'd be fine keeping the tree local-only, and exposing only a literal output string over the API. There's precedence for this with things like `podman top`)

Reply:
(Or was it `podman stats`... I forget which)